snippets/vectorAngle.md

```diff
@@ -0,0 +1,36 @@
+---
+title: vectorAngle
+tags: math,intermediate
+---
+
+Returns the angle (theta) value between two vectors.
+
+Use `Math.trunc()` to pick the integer value of length. Use `slice()` slice the incoming array into individual vectors. 
```

Comment:
- `length` should be inside backticks as it references a variable.
- `Array.prototype.slice()` instead of `slice()`.
- Each sentence should start in a new line.
- Phrasing on either sentence is unclear. Please rephrase.